plugin/storage/cassandra/spanstore/operation_names_test.go

```diff
@@ -160,17 +167,14 @@ func TestOperationNamesStorageGetServices(t *testing.T) {
 			query.On("Iter").Return(iter)
 
 			s.session.On("Query", mock.AnythingOfType("string"), mock.Anything).Return(query)
-			services, err := s.storage.GetOperations("service-a")
+			services, err := s.storage.GetOperations(&spanstore.OperationQueryParameters{ServiceName: "service-a"})
```

Comment:
Weird, this line of code is within the vertical 120 char line in my local intelliJ, somehow github still shows scroll bar.